### PR TITLE
TwiML generator

### PR DESCRIPTION
**Pull request by jingming**

### CODE REVIEW

**Comment by dougblack** [comment]
This seems good. The `setAttribute()` helper will be very useful.

**Comment by dougblack** [comment]
This seems good. The `setAttribute()` helper will be very useful.

File: Twilio/TwiML/Fax/Receive.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+     * @return TwiML $this.
+     */
+    public function setAction($action) {
+        return $this->setAttribute('action', $action);
+    }
+
+    /**
+     * Add Method attribute.
+     * 
+     * @param http_method $method Receive action URL method
```
Comment (dougblack) [inline Twilio/TwiML/Fax/Receive.php]: nit: is there any built in support for having things like this show up as camelCase to match surrounding code?

Comment (jingming) [reply Twilio/TwiML/Fax/Receive.php]: sure!

**Pull request closed without merging**